packages/skeleton-loader/src/ListSkeleton/ListSkeleton.styles.ts

```diff
@@ -0,0 +1,31 @@
+import { css } from '@leafygreen-ui/emotion';
+import { spacing } from '@leafygreen-ui/tokens';
+
+export const skeletonListWrapperStyles = css`
+  width: 100%;
+  padding: 0;
+  margin: 0;
+`;
+
+export const getSkeletonListItemStyles = (
+  index = 0,
+  bulletsOnly?: boolean,
+) => css`
+  list-style: none;
+
+  &:not(:first-child),
+  &:not(:last-child) {
```

Comment:
this still targets the first and last element, is that what you want?

Reply:
Ah, I don't actually need this anymore since I'm using margin. (was using padding previously)